packages/xforms-engine/src/parse/model/RootAttributeDefinition.ts

```diff
@@ -1,38 +1,59 @@
 import { XMLNS_NAMESPACE_URI } from '@getodk/common/constants/xmlns.ts';
-import type { NamedNodeDefinition } from '../../lib/names/NamespaceDeclarationMap.ts';
+import type { StaticAttribute } from '../../integration/xpath/static-dom/StaticAttribute.ts';
+import {
+	NamespaceDeclarationMap,
+	type NamedNodeDefinition,
+} from '../../lib/names/NamespaceDeclarationMap.ts';
 import { QualifiedName } from '../../lib/names/QualifiedName.ts';
 import { escapeXMLText } from '../../lib/xml-serialization.ts';
+import type { BindDefinition } from './BindDefinition.ts';
+import { NodeDefinition } from './NodeDefinition.ts';
 import type { RootDefinition } from './RootDefinition.ts';
 
 interface RootAttributeSource {
 	readonly qualifiedName: QualifiedName;
 	readonly value: string;
 }
 
-/**
- * @todo This class is named and typed to emphasize its intentionally narrow
- * usage and purpose. It **intentionally** avoids addressing the much broader
- * set of concerns around modeling attributes in primary instance/submissions.
- */
-export class RootAttributeDefinition implements NamedNodeDefinition {
+export class AttributeDefinition
```

Comment:
I like the new name `AttributeDefinition`; `root` was too generic. Can this file adopt the name from this class? Not sure if there's any conflict.